server_py/flatgov/common/elastic_load.py

```diff
@@ -60,9 +60,14 @@ def indexBill(bill_path: str=PATH_BILL):
   else:
     dublinCore = ''
   dctitle = getText(billTree.xpath('//dublinCore/dc:title', namespaces={'dc': 'http://purl.org/dc/elements/1.1/'}))
-  billCongressTypeNumberVersion = '' 
+  billCongressTypeNumberVersion = ''
+  billVersion = ''
   if dctitle and dctitle.find(':') > -1:
```

Comment:
Why don't we put the try/except under the `if` statement here?
```python
doc_id = ''
if dctitle and dctitle.find(':') > -1:
  try:
    billVersion = dctitle.split(':')[0].split(' ')[-1].lower()
    doc_id = billnumber_text + billVersion
  except Exception:
    pass
    ```
    
    Then later:
    ```python
    if doc_id != '' and len(doc_id) > 7:
      doc['id'] = doc_id
   ```python

Reply:
That's a good idea.